### PR TITLE
[MilkmApp] Add uniswap v2 univ2 price checker pt 1

### PR DESCRIPTION
**Pull request by yvesfracari**

### CODE REVIEW

**Review by ribeirojose — DISMISSED** [review]
LGTM 👍  left a few ideas in https://github.com/bleu-fi/monorepo/pull/462/files, lmk what you think

File: apps/cow-tools/src/lib/priceCheckers.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      [goerli.id]: "0xEB2bD2818F7CF1D92D81810b0d45852bE48E1502",
+    },
+    arguments: [{ name: "minOut", type: "uint256" }],
+    name: PRICE_CHECKERS.FIXED_MIN_OUT,
+  },
+  [PRICE_CHECKERS.UNI_V2]: {
+    addresses: {
+      [goerli.id]: "0x5d74aFFFd2a0250ABA74D6703Bd8e140534b3F36",
+    },
+    arguments: [{ name: "allowedSlippageInBps", type: "uint256" }],
```
Comment (luizakp) [inline apps/cow-tools/src/lib/priceCheckers.ts:23]: 👌

File: apps/cow-tools/src/lib/priceCheckers.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+): `0x${string}` {
+  const { arguments: priceCheckerArgs } = priceCheckerInfoMapping[priceChecker];
+  if (priceCheckerArgs.length !== args.length) {
+    throw new Error(`Invalid number of arguments for ${priceChecker}`);
+  }
+  return encodePacked(
+    priceCheckerArgs.map((arg) => arg.type),
+    args,
+  );
+}
```
Comment (luizakp) [inline apps/cow-tools/src/lib/priceCheckers.ts:40]: 🔝

File: apps/cow-tools/src/lib/transactionFactory.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    amount,
+    priceChecker,
+    args,
+  }: MilkmanOrderArgs): BaseTransaction {
+    const priceCheckerInfo = priceCheckerInfoMapping[priceChecker];
+    const priceCheckerAddress = priceCheckerInfo.addresses[goerli.id];
+    const priceCheckerData = encodePriceCheckerData(priceChecker, args);
+    return {
+      to: MILKMAN_ADDRESS,
+      value: "0",
```
Comment (luizakp) [inline apps/cow-tools/src/lib/transactionFactory.ts:79]: value is always 0?

**Review by luizakp — APPROVED** [review]
👍

**Pull request merged**